tools/rotate-certs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+namespace=openshift-cnv
+_kubectl="${KUBECTL_BINARY:-oc}"
+
+
+if ! options=$(getopt -o n: -- "$@")
+then
+    exit 1
+fi
+
+eval set -- "$options"
+
+while true; do
+    case "$1" in
+    -n)
+        shift; # The arg is next in position args
+        namespace=$1
+        ;;
+    --)
+        shift
+        break
+        ;;
+    esac
+    shift
+done
+shift $((OPTIND-1))
+
+echo "# Rotating kubemacpool certificates ..."
+${_kubectl} --namespace "${namespace}" delete pods -l app==kubemacpool
+
+echo "# Rotating cdi certificates ..."
+${_kubectl} delete secrets --namespace "${namespace}" -l cdi.kubevirt.io
```

Comment:
@djzager cc.  PR https://github.com/kubevirt/hyperconverged-cluster-operator/pull/333 makes changes to the labels.  Should they change?

Reply:
We don't create any secrets so they shouldn't be impacted but I'm looking into the impact of the label changes now.

Reply:
Could you clarify what you mean by "changes to the labels"? I don't think that you can change these labels, since that would mean that you change labels on objects which are created by the operators managed by hco.

Reply:
@djzager my point in mentioning it is you're changing the labels, but there's an expectation to still use the "old" labels e.g. `kubevirt.io = ""`.  So while it may not cause a problem in this script, it could cause a problem for us somewhere else.   I want to make sure we cover our bases.

@rmohr here's an example: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/333/files#diff-412c4fc972d3983c4e02e20a248c6421L15 .

Reply:
Looks like this changes the labels of operator-level objects. Let me check if that may be an issue for SSP.

Reply:
When these operators are deployed using OLM. You don't have the luxury of setting those labels.

All of the labels I see that are impacted in #333 are ones that you can't set when using OLM anyway. I'm looking for one where this is not the case.

Reply:
Everything that we can get from a component operator's CSV should be safe (ie. https://github.com/kubevirt/hyperconverged-cluster-operator/blob/fcb0bbc05213c6967fc30228828e06c41721a19e/deploy/olm-catalog/kubevirt-hyperconverged/0.0.3/kubevirt-hyperconverged-operator.v0.0.3.clusterserviceversion.yaml#L1444-L1460 )

The things that we can get from each component's CSV include the Deployment Spec, (Cluster)Rules, and Service Account Names so the fact that we don't have a `"kubevirt.io: ""` label on the `cdi-operator` service account should not be a breaking change.

Hope that helps.